docs/content/any/getting-started/_index.md

```diff
@@ -1,11 +1,44 @@
 ---
-# TODO: Rename?
-title: "Getting Started"
+title: "Get Started"
 weight: 1
-any: false
-below_picker: roles-cta
+any: true
 ---
 
-# Jump in
+# Get Started
 
-Learn the basics of using Oso and get Oso set up in your application.
+Let's get started with Oso! Oso is a framework for application
+authorization with built-in primitives for modeling authorization. Oso
+consists of the following components:
+
+- **Oso Library**: Oso is a library that you include in your application to
+enforce authorization. The library supports multiple languages,
+currently Python, Node, Ruby, Go, Java, and Rust.
+- **Polar Policy Language**: With Oso, you express authorization logic
+declaratively using Polar, our policy language. The Oso library
+evaluates policies to make authorization decisions. Polar policies are
+written directly over the same data types that you use in your
+application.
+    - **Resources, Roles and Permissions**: The most common way to model
+    authorization with Oso is through resources. Specify resources in
+    Polar and the roles and permissions you want users to have on them.
+    - **Rules**: Since Polar is a declarative language, you can extend
+    the resource model to cover other cases as needed by your
+    application. For example, you may deny access by banned users or
+    allow any user to access a public resource.
+{{% ifLang "python" %}}
+- **Framework Libraries**: Oso has pre-built integrations for web
```

Comment:
not sure what 'pre-built' means in this context

```suggestion
- **Framework Libraries**: Oso has integrations for web
```

maybe 'out-of-the-box'? although technically it's a different box (e.g., `sqlalchemy-oso` vs. `oso`)